app/views/menus/show.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1= "Menu for #{@facade.menu.day_stamp}"
+%h2= "#{t :order_list}:"
```

Comment:
You're using `t('key')` and `t :key` forms of calling i18n. Just FYI it would be a hell to grep some keys when needed. Not an important issue though.